test/labeling/apply/test_lf_applier.py

```diff
@@ -9,20 +9,16 @@
 
 from snorkel.labeling.apply import LFApplier, PandasLFApplier
 from snorkel.labeling.lf import labeling_function
-from snorkel.labeling.preprocess import Preprocessor
+from snorkel.labeling.preprocess import preprocessor
 from snorkel.labeling.preprocess.nlp import SpacyPreprocessor
 from snorkel.types import DataPoint, FieldMap
 
 
-class SquarePreprocessor(Preprocessor):
-    def __init__(self, x_field: str, squared_x_field: str) -> None:
-        super().__init__(dict(x=x_field), dict(x=squared_x_field))
+@preprocessor(dict(x="a"), dict(x="a"))
+def square(x: float) -> FieldMap:
+    return dict(x=x ** 2)
```

Comment:
Two requests:

1. Make the tests more readable with (a) comments and (b) readable attribute names (e.g. this should be `@preprocessor(dict(x="num"), dict(x="num_squared"))`.  I think improving test readability is *very* critical to team development speed

2. It would be really nice if you could provide only one attribute, and this would default to modifying the attribute in place